phpBB/styles/prosilver/theme/colours.css

```diff
@@ -697,6 +697,15 @@ dd.profile-warnings {
 	color: #d41142;
 }
 
+.button[disabled],
+.button[disabled]:hover,
+.button.disabled,
+.button.disabled:hover {
+	background: #e0e0e0;
+	border-color: #9e9e9e;
+	color: #9e9e9e;
+}
+
```

Comment:
Probably dumb question but most of this is a duplication of `buttons.css` so the latter will be overriden with these ones as in `stylesheet.css` it goes later than `buttons.css`.

Reply:
I have removed the duplicated css now